docs/ValidationCriteriaMaster.md

```diff
@@ -0,0 +1,762 @@
+| ------------------- | ------------------------ | -------------------- | ----------------------- | ---------------------------- | ---------------------------- | ---------------- |
+| IfcFacilityPart     | LEVELCROSSING            | ROAD_LC              | IfcFacilityPart         | LEVELCROSSING                | RW_LC                        | Crosses          |
+
+</details>
+
+<details><summary>Product Relative Positioning Table template</summary>
+
+> OPTION 1: Use this template if you need to check at predefined type or object type level
+
+| **Product**     | **Product Type** | **Size** | **Positioning Element** | **Positioning Element Type** |
+|---------------- |------------------|----------|-------------------------|------------------------------|
+| IfcRail         | RAIL             | 2        | IfcReferent             |       STATION                |
+
+
+> OPTION 2: Use this template if you need to check at the individual instance level
+
+| **Product**     | **Product Type** | **Product Name** | **Size** | **Positioning Element** | **Positioning Element Type** | **Element Name** |
+|-----------------|------------------|------------------|----------|-------------------------|------------------------------|------------------|
+| IfcRail         | RAIL             | RAIL-01          | 2        | IfcReferent             | na                           |   Alignment-1 Primary Route               |
+| IfcTrackElement | SLEEPER          | Sleeper-01       | 1        | IfcReferent             | STATION                      |   PA+220         |
+
+</details>
+
+<details><summary>Product Geometric Representation Table template</summary>
+
+> OPTION 1: Use this template if you need to check at predefined type or object type level
+
+| **Product**     | **Product Type** | **Representation Identifier** | **Representation Type** |**Items**           |
+|-----------------|------------------|-------------------------------|-------------------------|--------------------|
+| IfcAlignment    | na               | Axis                          |       Curve3D           |1 IfcGradientCurve  |
+
+
+> OPTION 2: Use this template if you need to check at the individual instance level
+
+| **Product**     | **Product Type** | **Product Name**   | **Representation Identifier** | **Representation Type** |**Items**           |
+|-----------------|------------------|--------------------|-------------------------------|-------------------------|--------------------|
+| IfcAlignment    | na               | Alignment Primary_1| Axis                          |       Curve3D           |1 IfcGradientCurve  |
+
+**NOTE**:
+- Items should be listed in the cell with their number.
+- One shape representation for a product should be documented in one row. If there are multiple representations, they should be documented in multiple rows.
+
+</details>
+
+<details><summary>Product Placement Table template</summary>
+
```

Comment:
Where can we specify the actual linear- or local placement parameters (RelativePlacement, IfcLInearPlacement.CartesianPoint)?

Reply:
I think this requires another table in this section. Do you already have an example for it?

Reply:
I don't know if it is enough, but for linearly placed level crossing equipment, I just put something like this in a single column:
Linear placement (A.a.A1.IfcGradientCurve, DistanceAlong=1715, OffsetLateral=0, OffsetVertical=-0.221, _CartesianPosition=x,y,z_). I did not add any axis2placement though...